java/connector-node/connector-api/src/main/java/com/risingwave/connector/api/sink/SinkFactory.java

```diff
@@ -22,4 +22,9 @@ public interface SinkFactory {
     SinkWriter createWriter(TableSchema tableSchema, Map<String, String> tableProperties);
 
     void validate(TableSchema tableSchema, Map<String, String> tableProperties, SinkType sinkType);
+
+    default SinkCoordinator createCoordinator(
```

Comment:
Seems no connector support the coordinator yet? And when create a sink, how to specify the usage of the coordinator mechanism?

Reply:
For sink without coordinator, it is not expected to call `createCoodinator`, so the default implementation is to throw an exception. For sink supporting coordinator, the meta node will finally call the `create_coordinator` of the sink. Therefore, to specify that the sink has coordinator, we only need to implement the `createCoordinator` method.